Formula/o/odin.rb

```diff
@@ -39,8 +133,6 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/odin version")
```

Comment:
Worth keeping this in too, I think.

Reply:
I removed it because it fails with the `--head` flag, I can add it back in

Reply:
Ok, thanks.